assets/src/edit-story/app/layout/index.js

```diff
@@ -58,7 +63,7 @@ const Area = styled.div`
 
 function Layout() {
   return (
-    <Editor>
+    <Editor role="main" aria-label={__('Web Stories Editor', 'web-stories')}>
```

Comment:
If `Editor` is a `styled.section`, this `role` attribute will not be necessary.

Source: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Region_role

> Developers should always prefer using the correct semantic HTML element, in this case `<section>`, over using ARIA.

Reply:
Attention ⚠️ 

**The main landmark role should only be used once per document.**

Source: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Main_role

In WordPress admin, the `#wpbody` div already has that role.

Reply:
Ok. Will check to region as well.